lib/api/src/Cardano/Wallet/Api/Types/Error.hs

```diff
@@ -293,6 +301,34 @@ data ApiErrorTxOutputLovelaceInsufficient = ApiErrorTxOutputLovelaceInsufficient
         via DefaultRecord ApiErrorTxOutputLovelaceInsufficient
     deriving anyclass NFData
 
+data ApiErrorOutputTokenQuantityExceedsLimit = ApiErrorOutputTokenQuantityExceedsLimit
+    { address
+        :: !Text
+    , policyId
+        :: !Text
+    , assetName
+        :: !Text
+    , quantity
+        :: !Natural
+    , maxQuantity
+        :: !Natural
+    }
+    deriving (Data, Eq, Generic, Show, Typeable)
+    deriving (FromJSON, ToJSON)
+        via DefaultRecord ApiErrorOutputTokenQuantityExceedsLimit
+    deriving anyclass NFData
+
+data ApiErrorOutputTokenBundleSizeExceedsLimit = ApiErrorOutputTokenBundleSizeExceedsLimit
+    { address
+        :: !Text
+    , bundleSize
+        :: !Int
```

Comment:
Request: replace this with `Natural`

Justification: it's impossible to have a bundle that is negatively-sized.

```suggestion
        :: !Natural
```

Reply:
done